### PR TITLE
[refs #DI-131] Email template

### PR DESCRIPTION
**Pull request by Android63**
Email template for WP HTML Mail plugin to use Nightingale styling.

A test email looks like this:
Screen Shot 2019-03-22 at 12 42 06

### CODE REVIEW

File: wp-html-mail/template.html [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+							</td>
+						</tr>
+						<tr>
+							<td align="center" valign="top" style="font-size:1px; line-height:1px;" class="w100">
+								<img src="https://i7.createsend1.com/ti/j/E9/7D5/33E/020629/spacer.gif" width="1" height="10" alt="" border="0" style="display:block;" />
+							</td>
+						</tr>
+						<tr>
+							<td class="footer" align="center" valign="top" style="font-size: 10px; line-height: 16px; font-family: Frutiger, Arial, sans-serif; color: #425563;" class="w100">
```
Comment (marcinca) [inline wp-html-mail/template.html]: class defined twice

File: wp-html-mail/template.html [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+			.webversion a{text-decoration:underline; color:#213F20 !important;}
+			.footer a {font-size: 12px; !important;}
+			/*Responsive css*/
+			@media only screen and (max-width: 600px) {
+				*[class=gmailfix] {width:100% !important; min-width:1px !important;}
+				*[class=border] {width:90% !important; margin:0 auto; border-left: 1px solid #D8D6D6; border-right: 1px solid #D8D6D6;}
+				*[class=wrapper] {width:100% !important; margin:0 auto;}
+				*[class=inner] {width:90% !important; margin:0 auto; max-width: 560px !important;}
+				*[class=w100] {width:100% !important; margin:0 auto;}
+				*[class=img100] {width:100% !important; height:auto !important;}
```
Comment (marcinca) [inline wp-html-mail/template.html]: All classes below to `section-border-mid` seems to be not used in this email

File: wp-html-mail/template.html [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+		<meta http-equiv="content-language" content="en">
+		<title>NHS Leadership Academy</title>
+		<meta content="width=device-width, initial-scale=1.0" name="viewport" />
+        <style type="text/css">
+			body{width:100% !important; -webkit-text-size-adjust:none; margin:0; padding:0;}
+			.ReadMsgBody{width:100%;}
```
Comment (marcinca) [inline wp-html-mail/template.html]: This selector class `ReadMsgBody` is never used in the following code, same `ExternalClass` below

File: wp-html-mail/template.html (hunk 1 above)
Comment (Android63) [reply wp-html-mail/template.html]: Corrected

File: wp-html-mail/template.html (hunk 2 above)
Comment (Android63) [reply wp-html-mail/template.html]: I've removed all unused classes

File: wp-html-mail/template.html (hunk 3 above)
Comment (Android63) [reply wp-html-mail/template.html]: All unused classes are now gone

**Pull request merged**